### PR TITLE
BETA: Update FCB datasheet link

### PR DESCRIPTION
**Pull request by Caleb-Ellis**
## Done

- Updated Foundation Cloud Build datasheet link

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/openstack/consulting
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Check that the datasheet links take you to https://assets.ubuntu.com/v1/b71c0af7-canonical-foundation-cloud-build-2018-08-09.pdf

### CODE REVIEW

File: templates/openstack/consulting.html [hunk 1]
```diff
@@ -13,7 +13,7 @@ <h2 class="p-heading--three">Carrier grade, banking security, government scale.
         <p>Hundreds of successful OpenStack clouds underpin our consulting, training, architecture design, and hardware procurement advice - to help you evaluate and deliver OpenStack.</p>
         <p>
           <a href="/openstack/contact-us?product=foundation-cloud" onclick="dataLayer.push({'event' : 'GAEvent', 'eventCategory' : 'Internal Link', 'eventAction' : 'Engage Canonical exports', 'eventLabel' : 'Engage Canonical exports - Top CTA', 'eventValue' : undefined });" class="p-button--positive">Engage Canonical experts</a>
-          <a href="{{ ASSET_SERVER_URL }}6a031fb9-EN_Foundation-Cloud-Build_screen-AW_19.07.18.pdf" class="p-button--neutral">Read our consulting datasheet</a>
+          <a href="{{ ASSET_SERVER_URL }}6e65055c-EN_Foundation-Cloud-Build_screen-AW_07.08.18.pdf" class="p-button--neutral">Read our consulting datasheet</a>
```
Comment (pmahnke) [inline templates/openstack/consulting.html]: sorry, but can we rename this to Foundation-Cloud-Build-2018-08.pdf and update the spreadsheet

**Comment by Caleb-Ellis** [comment]
@pmahnke Updated with new file and filename

**Comment by pmahnke** [comment]
@Caleb-Ellis did you rename it differently on purpose?

**Comment by Caleb-Ellis** [comment]
@pmahnke I was following the filenames practices https://github.com/canonical-webteam/practices/blob/master/content/filenames.md

**Comment by pmahnke** [comment]
@Caleb-Ellis fair enough, but you got the month and day in the wrong order?

**Pull request merged**